rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -0,0 +1,110 @@
+- project:
+    name: 'RPC-Artifact_Build-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - artifacts:
+          branch: artifacts-14.0
+          branches: "artifacts-.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
+      - Python
+      - Container
+    ztrigger:
+      - pr:
+          CRON: ""
+          STAGES: "Allocate Resources, Connect Slave, Build {context} Artifacts, Cleanup"
```

Comment:
Apparently the JJB renders, so I guess it might: https://rpc.jenkins.cit.rackspace.net/job/Jenkins-Job-Builder/313/console